controller/backing_image_controller.go

```diff
@@ -88,23 +88,23 @@ func NewBackingImageController(
 		DeleteFunc: bic.enqueueBackingImage,
 	})
 
-	backingImageManagerInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
+	backingImageManagerInformer.Informer().AddEventHandlerWithResyncPeriod(cache.ResourceEventHandlerFuncs{
```

Comment:
Is this resync period only applied to these specific handlers? If YES, that would be great.

Reply:
> Yes, it is only applied to these specific handlers.

Some info from the code path:

1. Each AddEventHandlerWithResyncPeriod is adding a process listener to the sharedProcessor: [link](https://github.com/longhorn/longhorn-manager/blob/ad25434856d45c92af5dcbb4acd2e4413a4dd508/vendor/k8s.io/client-go/tools/cache/shared_informer.go#L476)
1. The sharedProcessor collects all the listeners that need to do resync and put them into syncingListeners: [link](https://github.com/longhorn/longhorn-manager/blob/ad25434856d45c92af5dcbb4acd2e4413a4dd508/vendor/k8s.io/client-go/tools/cache/shared_informer.go#L610-L622). The listeners that need to do resync are the ones with resyncPeriod > 0: [link](https://github.com/longhorn/longhorn-manager/blob/ad25434856d45c92af5dcbb4acd2e4413a4dd508/vendor/k8s.io/client-go/tools/cache/shared_informer.go#L762)
1. When it is the time to resync, the share sharedProcessor distribute the event to the listener that want to resync only: [link](https://github.com/longhorn/longhorn-manager/blob/ad25434856d45c92af5dcbb4acd2e4413a4dd508/vendor/k8s.io/client-go/tools/cache/shared_informer.go#L570-L583)